### PR TITLE
Added K8S 1.8-11.x and the missing -strict

### PR DESCRIPTION
**Pull request by consideRatio**
@garethr thank you for the work you have done with this and `kubeval`! It made me catch a sneaky bug that took a long time to find. I hope to use `kubeval` with an even more satisfactory feeling using these updates.

## This PR

1. Added fresh versions
    - `v1.11.0`
    - `v1.10.5`
    - `v1.9.9`
    - `v1.8.14`
2. Complemented versions lacking `-strict` directories
3. Updated master fully
4. I added tooling to avoid running old stuff once again pointlessly (`recreate=false` in `build.sh`)

### CODE REVIEW

**Comment by consideRatio** [comment]
@hiddeco thanks for paving the way, your PR functioned as documentation on how to contribute for me =D

File: build.sh [hunk 1]
```diff
@@ -81,10 +85,23 @@ declare -a arr=(master
...
-    openapi2jsonschema -o "${version}" --kubernetes --prefix "${prefix}" "${schema}"
+    if [ "${recreate}" = true ] || [ ! -d "${version}-standalone-strict" ]; then
+        openapi2jsonschema -o "${version}-standalone-strict" --kubernetes --stand-alone --strict "${schema}"
+    fi
+
+    if [ "${recreate}" = true ] || [ ! -d "${version}-standalone" ]; then
+        openapi2jsonschema -o "${version}-standalone" --kubernetes --stand-alone "${schema}"
+    fi
+
+    if [ "${recreate}" = true ] || [ ! -d -o "${version}-local" ]; then
```
Comment (willthames) [inline build.sh]: There is an extra `-o` here that is incorrect.

Comment (consideRatio) [reply build.sh]: Oh woops - I tested what would happen with this this mistake of including `-o` and realized that this means that whatever I ran with `recreate=false` with a non-existing `...-local` directory were never created.

I'll make sure to fix this, and generate potentially missing `...-local` directories.

---

```
# the empirical test I made to determine what adding -o caused
mkdir test
if [ ! -d -o "test" ]; then echo "1"; fi
if [ ! -d "test" ]; then echo "2"; fi
if [ ! -d -o "testNoExistance" ]; then echo "3"; fi
if [ ! -d "testNoExistance" ]; then echo "4"; fi
# output was "4" only
```

**Comment by consideRatio** [comment]
Thanks @willthames for reviewing the changes I made to the generation script.

I had generated all the `...-local` folders were before I introduced these conditionals with this bug causing non-existing `-local` folders to not be generated - in other words: no harm was done.

**Comment by willthames** [comment]
@consideRatio - yeah, I noticed because I wanted to generate `-local-strict` and copied the `-local` step and then wondered why nothing happened :)

**Comment by garethr** [comment]
Apologies for taking an age to get back to this, I've not had much time to maintain in a while unfortunately. I've done a bulk update of versions today, and have some plans afoot to move this into a standalone org and get more folks involved as maintainers. I'll ping when I have more details in case you might still be interested.

**Pull request closed without merging**